packages/release/mask-local-mnt.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Mask Local Mnt Directory (/local/mnt)
+DefaultDependencies=no
+RequiresMountsFor=/local /mnt
+BindsTo=mnt.mount
+ConditionPathIsMountPoint=!/local/mnt
+Conflicts=umount.target
+Before=umount.target
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+
+# Mask `/local/mnt` to avoid confusion, since it will have most of the contents
+# of `/mnt` but not any of the mounts.
+ExecStart=/usr/bin/mount --bind --options nosuid,nodev,noexec,private /srv /local/mnt
+ExecStop=/usr/bin/umount /local/mnt
```

Comment:
If this is a `oneshot`, do you still need the `ExecStop` directive?

Reply:
This guards against the case where `/mnt` is unmounted.

It should never happen in practice, but if it did, then if `/mnt` were mounted again via the `mnt.mount` unit, it might bind-mount the `/srv` mount over `/mnt`, instead of bind-mounting the underlying directory `/local/mnt`.